src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -8,6 +8,9 @@
 
 #include "DataStructures/Tensor/TypeAliases.hpp"
 
+/// \ingroup GeneralRelativityGroup
```

Comment:
remove 2-line comment (the namespace has already been documented)

Reply:
Removed